domain/apiresponses/responses.go

```diff
@@ -20,8 +20,10 @@ import "github.com/pivotal-cf/brokerapi/v8/domain"
 type EmptyResponse struct{}
 
 type ErrorResponse struct {
-	Error       string `json:"error,omitempty"`
-	Description string `json:"description"`
+	Error            string `json:"error,omitempty"`
+	Description      string `json:"description"`
+	InstanceUsable   bool   `json:"instance_usable,omitempty"`
```

Comment:
- The effect of `omitempty` is that the value will only appear in the JSON output when it is true. We think these should be `*bool` type which allows the value in Go to be `nil`, `true` or `false`; resulting in the JSON being omitted when the value is `nil` and is otherwise `true` or `false`.
- The OSBAPI spec says that `instance_usable` and `update_repeatable` should only appear in relevant errors, so the ability to control the value like this is needed.

Reply:
> Hi @tillknuesting. Thank you for this pull request.
> 
> I have been reviewing it with my colleague @FelisiaM. We noticed that the spec says these fields should also be present in the output for Last Operation. Also the tests should be updated to test for the new fields.
> 
> Please let us know your thoughts. We can provide more details if that would be helpful.
> 
> Best wishes,
> 
> @blgm and @FelisiaM

Thank you for your response, will have another look and take your suggestions into account.